modules/loader-utils/src/lib/request-utils/request-scheduler.ts

```diff
@@ -25,7 +26,10 @@ const DEFAULT_PROPS: Required<RequestSchedulerProps> = {
   // Specifies if the request scheduler should throttle incoming requests, mainly for comparative testing
   throttleRequests: true,
   // The maximum number of simultaneous active requests. Un-throttled requests do not observe this limit.
-  maxRequests: 6
+  maxRequests: 6,
+  // Specifies a debounce time, in milliseconds. All requests are queued, until no new requests have
```

Comment:
Nit: If we use JSDoc syntax for comments `/** ...  */` we can see these comments when hovering in vscode.

Reply:
Yes, I prefer that as well — have updated this and the other options.